Assets/Scripts/Features/UI/Ingame UI/PauseWindow.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PauseWindow : MonoBehaviour
+{
+    public void CloseWindow()
```

Comment:
Same goes for this function, no XML commands have been used to describe a public function.